app/namespace.go

```diff
@@ -1489,26 +1417,32 @@ func NewNamespaceCommand(getNamespaceClientFn GetNamespaceClientFn) (CommandOut,
 				RequestIDFlag,
 			},
 			Action: func(ctx *cli.Context) error {
-				namespaceName := ctx.String(NamespaceFlagName)
+				namespaceName := ctx.String(NamespaceFlag.Name)
 				sinkName := ctx.String(sinkNameFlag.Name)
-				resourceVersion, err := c.getExportSinkResourceVersion(ctx, namespaceName, sinkName)
-				if err != nil {
-					return err
+				resourceVersion := ctx.String(ResourceVersionFlag.Name)
+
+				if resourceVersion == "" {
+					getExportSinkRes, err := c.cloudAPIClient.GetNamespaceExportSink(c.ctx, &cloudservice.GetNamespaceExportSinkRequest{
+						Namespace: namespaceName,
+						Name:      sinkName,
+					})
+					if err != nil {
+						return fmt.Errorf("unable to get export sink: %v", err)
+					}
+					resourceVersion = getExportSinkRes.GetSink().GetResourceVersion()
 				}
 
-				deleteRequest := &namespaceservice.DeleteExportSinkRequest{
+				deleteRequest := &cloudservice.DeleteNamespaceExportSinkRequest{
 					Namespace:       namespaceName,
-					SinkName:        sinkName,
+					Name:            sinkName,
 					ResourceVersion: resourceVersion,
-					RequestId:       ctx.String(RequestIDFlagName),
```

Comment:
do we not support sending the request id anymore?

Reply:
Yes. request id is not using any more. We're using AsyncOperationId instead